R/char_select.R

```diff
@@ -44,7 +44,7 @@ char_select.character <- function(x, pattern, selection = c("keep", "remove"),
     
     if (is.dictionary(pattern))
         pattern <- unique(unlist(as.list(pattern), use.names = FALSE))
-    if (is.collocations(pattern))
+    if ("collocations" %in% class(pattern))
```

Comment:
I think it is better to have object checking functions in the core package at least as internals.

Reply:
Agreed. Added back in c66f617.